indico/modules/events/papers/blueprint.py

```diff
@@ -26,7 +26,8 @@
 
 
 # Display pages
-_bp.add_url_rule('/papers/', 'call_for_papers', lambda **kw: 'TODO')
+_bp.add_url_rule('/papers/', 'call_for_papers', display.RHCallForPapers)
+_bp.add_url_rule('/papers/select_contribution/', 'select_contribution', display.RHSelectContribution)
```

Comment:
We use dashes in URLs instead of underscores (also, don't put a trailing slash if there's nothing "inside")